_includes/teammember.html

```diff
@@ -0,0 +1,38 @@
+<!-- Modal für jedes Teammitglied -->

```

Comment:
Die Modals verwenden wir aber nicht mehr, oder? In dem Fall den Code bitte löschen.